docker-compose.yml

```
@@ -1,10 +1,10 @@
 services:
   api: 
-    build: /api/
+    build: ./api
     ports:
-      -5000:5000
+      - "5000:5000"
     volumes:
-      - /api : app/api
+      - ./api:/app/api
     environment:
       FLASK_DEBUG: 1
     command: flask run -h 0.0.0.0
```

```suggestion
    environment:
      FLASK_DEBUG: 1
    command: flask run -h 0.0.0.0
    depends_on:
      - db
    restart: unless-stopped
```

just for good practice, it ensures db starts before api and second for restarting during crashes in dev, also good for other containers too